### ISSUE TITLE
onPrintDialogClose issue

### ISSUE THREAD

**Issue by ektaraval** [crabbly/Print.js#276]
I am printing a PDF but once the print dialog box is closed, I need to redirect my page. I used  onPrintDialogClose which is redirecting the page to desired location but before the print dialog is closed.
Here is my code. What I need to do is when some one click on "Continue" button it should display the print dialog for test.pdf when clicked on "Print" button it should print the pdf and redirect to PageName.cfm file.

<button type="button" OnClick="printJS({printable:'test.pdf', type:'pdf', onPrintDialogClose: 'pageRedirect()' })"> Continue </button>

function pageRedirect(){
        window.location.href= 'PageName.cfm'
}

**Comment by crabbly** [comment]
Hi @ektaraval ,

Please post the `printJS` usage as well. Thank you.

**Comment by crabbly** [comment]
Also, is this happening on all browsers? Post your OS as well. Thx.

**Comment by crabbly** [comment]
Hey @ektaraval, I couldn't find anything wrong with the `onPrintDialogClose` parameter. I created a codesandbox showing how you should use it.

https://codesandbox.io/s/r71k74qq7m

Feel free to reopen this if necessary. Thx.

**Comment by jimmy84** [comment]
I experience the similar issue with google chrome, this happened when **showModal: true** is present.

my sample code below
```
printJS({
	type: 'pdf',
	showModal: true,
	printable: "http://localhost/getpdf/test",
	onPrintDialogClose: function () {
		window.location.href = "http://google.com"
	}
});
```

**Comment by crabbly** [comment]
Improvements to the `onPrintDialogClose` callback have been published to npm with v1.5.0. It fixes issues when running chrome on windows.
Please run npm update print-js to get it. Thank you

**Comment by xkubow** [comment]
It still not working propertly. When the print dialog is closed (cancel button) the event is not triggerret. It trriggert after next page render . It is posible to simulate on https://jsfiddle.net/crabbly/oqea0rLm/.
